tmp/lanelet2_extension/CMakeLists.txt

```
@@ -62,6 +62,14 @@ target_link_libraries(autoware_lanelet2_validation
   lanelet2_extension_lib
 )
 
+ament_auto_add_executable(check_row src/check_row.cpp)
```

`check_row` might be confusing for most people. I don't mind the abbreviation in the code, but I think the name where user might see should be self-explanatory. We can also use auto completion in command line as well so it shouldn't be a big problem to have a long name.

```suggestion
ament_auto_add_executable(check_right_of_way src/check_right_of_way.cpp)
```